File: poll/sharpteam/SharpTeam.PollClient/Model/PollSession.cs
using System;
using System.Collections.Generic;
using System.Text;
using System.Xml;
using System.Xml.Serialization;

namespace Ilsrep.PollApplication.Model
{
    [XmlRoot("pollsession"), Serializable]
    public class PollSession
    {
        [XmlAttribute("id")] public int id;
        [XmlAttribute("name")] public string name;
        [XmlAttribute("testMode")] public bool testMode;
        [XmlAttribute("minScore")] public double minScore;
        [XmlArray("polls"), XmlArrayItem("poll", typeof(Poll) )] public List<Poll> polls = new List<Poll>();
    }
}

File: trunk/poll/jteam/src/ilsrep/common/Pollsession.java
package ilsrep.common;

import java.util.List;

import javax.xml.bind.annotation.XmlElementRef;
import javax.xml.bind.annotation.XmlRootElement;
import javax.xml.bind.annotation.XmlAttribute;

/**
 * The "polls" element.
 * 
 * @author Taras Kostiak
 * 
 */
@XmlRootElement(name = "pollsession")

public class Pollsession {
private String minScore="-1", name,id,testMode="false";
    /**
     * "poll" elements.
     */
    protected List<Poll> polls = null;

    /**
     * @see #polls
     */
      @XmlAttribute(name = "minScore")
    public String getMinScore() {
        return minScore;
    }
    public void setMinScore(String minScore) {
        this.minScore =minScore;
    }
    @XmlAttribute(name = "name")
   public String getName() {
        return name;
    }
    public void setName(String name) {
        this.name =name;
    }
    @XmlAttribute(name = "id")
   public String getId() {
        return id;
    }
    public void setId(String id) {
        this.id =id;
    }
    @XmlAttribute(name = "testMode")
   public String getTestMode() {
        return testMode;
    }
    public void setTestMode(String testMode) {
        this.testMode =testMode;
    }

    @XmlElementRef
    public List<Poll> getPolls() {
        return polls;
    }

    /**
     * @see #polls
     */
    public void setPolls(List<Poll> polls) {
        this.polls = polls;
    }

    
}
